packages/components/src/Actions/Action/Action.component.js

```diff
@@ -101,7 +103,11 @@ function Action(props) {
 	});
 
 	const buttonContent = getContent(props);
-
+	if (available === false) {
+		buttonProps.className = classnames({
+			hidden: available === false,
+		}, buttonProps.className || '');
```

Comment:
I see that as a possible problem, because the button is rendered in the DOM and it is clickable (programmatically). Or just by disabling the css property.
IMO, this responsibility should be managed by the container, rendering the Action or not, instead of letting the component do it with a flag.